libraries/SRV_Channel/SRV_Channel.cpp

```diff
@@ -122,13 +131,18 @@ void SRV_Channel::calc_pwm(int16_t output_scaled)
     } else {
         pwm = pwm_from_range(output_scaled);
     }
+    // If not in unlimited ouput we contrain the ouput
+    if (unlimited_pwm_mask & (1U << ch_num)) {
```

Comment:
I copy the logic of have_pwm_mask so it will be constraint if the mask is at 1. no?